gulpfile.js

```diff
@@ -129,7 +131,21 @@ defineBuildTask('HTMLImports');
 defineBuildTask('ShadowDOM');
 defineBuildTask('MutationObserver');
 
-gulp.task('build', ['webcomponents', 'webcomponents-lite', 'CustomElements', 
+gulp.task('CustomElementsV1', function () {
+  return gulp.src('./src/CustomElements/v1/CustomElements.js', {base: './'})
+      .pipe(closureCompiler({
```

Comment:
> We will distribute the built files.

You can still do that even if it's written with ES6 modules. I don't remember why, but I tried to get this running much earlier on when you started the work on this, but I couldn't get it working and I figured it was because you didn't hook it into your build system, and I didn't want to spend the time to figure it out. It may not be the case anymore if this file runs completely standalone.

If I can import this file directly into my ES6 environment, then I'll be happy (because that is the way in which JavaScript is recommended to be used now, while global-style code is discouraged though not disallowed).

In fact, I believe now that ES6 modules are official, it'd be nice if all new APIs were `import`able and never global. In this case, `customElements` would not be global, it would be imported. `import customElements from 'window/CustomElements'` or similar. This would be awesome and is the direction everything should go in.